File: SingleAPIExample/CSharp/Cancel.cs
using Splitit.SDK.Client.Api;
using Splitit.SDK.Client.Model;

namespace Examples
{
    public class CancelExample
    {
        InstallmentPlanApi PlanApi { get; }

        public void Cancel(string installmentPlanNumber)
        {
            var request = new CancelInstallmentPlanRequest
            {
                InstallmentPlanNumber = installmentPlanNumber,
                RefundUnderCancelation = RefundUnderCancelation.NoRefunds,
            };

            var cancelResponse = PlanApi.InstallmentPlanCancel(request);
        }
    }
}

File: SingleAPIExample/PHP/Cancel.php
<?php
$sessionId = '{{SessionID}}';
\SplititSdkClient\Configuration::sandbox()->setApiKey('{{APIKey}}');

$apiInstance = new SplititSdkClient\Api\InstallmentPlanApi(\SplititSdkClient\Configuration::sandbox(), $sessionId);

$request = new \SplititSdkClient\Model\CancelInstallmentPlanRequest([
    'installment_plan_number' => '{{InstallmentPlanNumber}}',
    'refund_under_cancelation' => 'OnlyIfAFullRefundIsPossible'
]);

try {
    $result = $apiInstance->installmentPlanCancel($request);
    print_r($result);
} catch (Exception $e) {
    echo 'Exception when calling InstallmentPlanApi->installmentPlanCancel: ', $e->getMessage(), PHP_EOL;
}
